eve/main.yml

```diff
@@ -133,12 +133,29 @@ stages:
               workloadPlane: 10.0.0.0/8
             END
             EOF
+      - ShellCommand:
+          # This step is needed so Kubelet registration doesn't fail for DNS
+          # entries longer than 63 characters
+          name: Change hostname to a shorter value
```

Comment:
minor: that's something you may want to put into the `eve/workers/openstack-single-node/requirements.sh` file instead

Reply:
Tracked [here](https://github.com/scality/metalk8s/issues/781#issuecomment-476565028).